spotify/spotify.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Add ~/.config to the XDG_CONFIG_HOME env var
```

Comment:
We dont add it here, it is substituted if the env var is empty, see https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Woops, I missed the "-" sign after the column somehow. Thanks!
Comment edited in https://github.com/christian-heusel/aur/pull/4/commits/184fdfba56fa9d1a3a2dcd4944b428397ac8d005